### PR TITLE
Build project docs by default

### PR DESCRIPTION
**Pull request by dpoirier**
* Add Sphinx configuration to docs subdir
* Add room in default docs organization for project-specific
  documentation
* Build docs on Travis

Branch: DJAN-88_RTD_by_default

### CODE REVIEW

**Comment by dpoirier** [comment]
I can't explain the Travis failures. I think given that one of the two tests is passing, things here are okay, but if someone else understands why the other test is failing, I'd love to know. Otherwise maybe we should just go ahead with this.

**Comment by dpoirier** [comment]
Let's see if JIRA notices when I mention DJAN-88 here.

File: Makefile [hunk 1]
```diff
@@ -82,8 +83,11 @@ update:
 	npm install
 	npm update
 
+# Build documentation
+html:
```
Comment (ironfroggy) [inline Makefile]: Can we call this `docs` instead of `html` which is a little ambiguous as part of a website' `Makefile`?

**Comment by ironfroggy** [comment]
Aside from the one comment, +1

File: Makefile (hunk 1 above)
Comment (dpoirier) [inline Makefile]: Makes sense, will do.

**Pull request merged**